deliver/lib/deliver/runner.rb

```diff
@@ -62,6 +66,19 @@ def upload_metadata
       UploadAssets.new.upload(options) # e.g. app icon
     end
 
+    def prepare_app_icons(options = {})
+      app_icon_path = options[:app_icon] if options[:app_icon]
+      app_icon_metadata = File.join(options[:metadata_path], "app_icon.png")
+      if app_icon_path.nil? && File.exist?(app_icon_metadata)
+        options[:app_icon] = app_icon_metadata
+      end
+      watch_icon_path = options[:apple_watch_app_icon] if options[:apple_watch_app_icon]
```

Comment:
Both code blocks seem overly complicated, can we use

```ruby
default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
options[:app_icon] ||= default_app_icon_path if File.exist?(default_app_icon_path)
```
This should have the same effect, right?